app/views/comments/_edit.html.erb

```diff
@@ -1,18 +1,17 @@
-<div class="comment-form-wrapper">
-  <div class="card card-body bg-light">
-    <form method="post" id="c<%= comment.id %>edit" style="display:none;" class="well" <% if comment.is_a?Answer %> action= "/answers/update/<%= comment.id %>" <% else %> action="/comment/update/<%= comment.id %>" <% end %>>
+<div class="card card-body bg-light comment-form-wrapper">
```

Comment:
I collapsed `<div class="card card-body bg-light">` and `<div class="comment-form-wrapper">` into one `<div>`. I know there's a lot of `<div>` spam in the views built up over the years, trying to do my part in reducing it.

I checked the CSS by both viewing locally and hunting through the stylesheets, and this doesn't seem to create any new issues.